tests/unit/AnnualReport.spec.ts

```diff
@@ -528,8 +547,7 @@ describe('Annual Report - Part 2A - Resuming with FAS staff payment', () => {
     const $route = { params: { filingId: '123' } } // draft filing id
     const wrapper = shallowMount(AnnualReport, { store, mocks: { $route }, vuetify })
     const vm = wrapper.vm as any
-    // wait for draft to be fetched
-    await flushPromises()
+    await flushPromises() // wait for draft to be fetched
```

Comment:
I did this so that when you search globally for "flushPromises", you get an idea how/why it's used right from the search results (ie, same line).